examples/pyfar_demo.ipynb

```diff
@@ -475,7 +475,7 @@
    "source": [
     "## Entering coordinate points<a class=\"anchor\" id=\"coordinates_enter\"></a>\n",
     "\n",
-    "Coordinate points can be entered manually or by using one of the available sampling schemes contained in `pyfar.samplings`. We will do the latter using an equal angle sampling and look at the information provided by the coordinates object:"
+    "Coordinate points can be entered manually."
```

Comment:
Maybe also add a note that samplings are available from spharpy?

Reply:
I think we need a general overview, where to find what. Maybe the website would be a good place.

Reply:
Using links in the respective documentations will help to some degree, but a short description on the website surely does help. It does not need to be a full listing of all functionality, that's part of the docs.